### ISSUE TITLE
[A11y] Work Show Page-All - Semantics (Emphasis)

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2028]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n
https://staging.oregondigital.org/concern/images/1g05fb620

Emphasis in "HOW TO CITE" is using `<i>`, but it should be using `<em>`. `<i>` has no semantic meaning, but `<em>` does.

### Suggested Fix

Use `<em>` for all italicized emphasis

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Check that `<i>`  and `<b>` are not used for emphasis.

**Comment by jechols** [comment]
Easy way to check if there are non-fontawesome `<i>` elements:

`Array.from(document.querySelectorAll('i')).filter(el => !el.classList.contains('fa'))`

**Comment by CGillen** [comment]
`<i>` already removed, not sure when
